shared/chat/conversation/messages/set-channelname/index.js

```diff
@@ -1,42 +1,14 @@
 // @flow
 import * as React from 'react'
-import {SmallUserNotice} from '../user-notice'
-import {ConnectedUsernames, Text} from '../../../../common-adapters'
-import {globalColors} from '../../../../styles'
-import {formatTimeForMessages} from '../../../../util/timestamp'
+import * as Types from '../../../../constants/types/chat2'
+import * as Kb from '../../../../common-adapters'
 
 type Props = {
-  author: string,
-  channelname: ?string,
-  onUsernameClicked: () => void,
-  setUsernameBlack: boolean,
-  timestamp: number,
+  message: Types.MessageSetChannelname,
 }
 
-const SetChannelname = (props: Props) => {
-  return (
-    <SmallUserNotice
```

Comment:
SmallUserNotice was used to emulate a message w/ author style. Instead we just treat those like messages w/ an author and remove all this code. Plus it'll be 100% the same as normal messages.
Note. This means that these rows now have timestamps on them, like the regular ones do. but design said this is preferred anyways